File: OLSource1/v1/VS_officedev/codesnippet/CSharp/creating-an-association-between-entities_4.cs
        public static IEnumerable<Contact> SalesOrderDetailToContact(int salesOrderID, int salesOrderDetailID)
        {
            const string ServerName = "MySQLServerName";
            AdventureWorksDataContext dataContext = new AdventureWorksDataContext
                  ("Data Source=" + ServerName + ";" +
                   "Initial Catalog=AdventureWorks;Integrated Security=True");
                
            int TempContactID = (from orders in dataContext.SalesOrderHeaders
                                 where orders.SalesOrderID == salesOrderID
                                 select orders.ContactID).Single();

            IEnumerable<Contact> contactList = from contacts in dataContext.Contacts
                                               where contacts.ContactID == TempContactID
                                               select contacts;
            return contactList;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/creating-an-association-between-entities_4.vb
    Public Shared Function SalesOrderDetailToContact(ByVal salesOrderID As Integer, ByVal salesOrderDetailID As Integer) As IEnumerable(Of Contact)
        Const ServerName As String = "MySQLServerName"
        Dim dataContext As AdventureWorksDataContext = _
            New AdventureWorksDataContext("Data Source=" & ServerName & _
                ";Initial Catalog=AdventureWorks;Integrated Security=True")

        Dim TempContactID As Integer = _
            (From orders In dataContext.SalesOrderHeaders _
            Where orders.SalesOrderID = salesOrderID _
            Select orders.ContactID).[Single]()

        Dim contactList As IEnumerable(Of Contact) = _
            From contacts In dataContext.Contacts _
            Where contacts.ContactID = TempContactID _
            Select contacts
        Return contactList

    End Function